allure-app-under-test/src/test/java/com/capgemini/ntc/selenium/projectY/SliderTest.java

```diff
@@ -0,0 +1,92 @@
+package com.capgemini.ntc.selenium.projectY;
```

Comment:
Add some blank lines to improve code readability.

Reply:
Done.